apps/api/src/certificate/controllers/certificate.controller.ts

```diff
@@ -0,0 +1,21 @@
+import assert from "http-assert";
+import { singleton } from "tsyringe";
+
+import { Protected } from "@src/auth/services/auth.service";
+import { CreateCertificateResponse } from "../http-schemas/create-certificate.schema";
+import { CertificateService } from "../services/certificate.service";
+
+@singleton()
+export class CertificateController {
+  constructor(private readonly certificateService: CertificateService) {}
+
+  @Protected([
+    { action: "sign", subject: "UserWallet" },
+    { action: "read", subject: "ApiKey" }
```

Comment:
I'm not sure I follow what this permission is for here

Reply:
Yea you're right it doesn't make sense and is redundant hehe